templates/k8s/quorum-deployment.yaml.erb

```diff
@@ -238,6 +238,9 @@ spec:
          <%- elsif @config["quorum"]["consensus"] == "istanbul" -%>
            args=\" --gcmode archive --istanbul.blockperiod 3 --syncmode full --mine --minerthreads 1 \";
            RPC_APIS=\"$RPC_APIS,istanbul\";
+         <%- end -%>
+         <%- if @config["quorum"]["quorum"]["Quorum_Version"] == "2.6.0" -%>
```

Comment:
Was debating that, but thought that we would have to explicitly support and check other issues when a new version became available. But I'll add it, as it will allow support for fixes and minor version changes.